webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/KnoraGroup.scala

```diff
@@ -34,6 +37,33 @@ final case class KnoraUserGroup(
   hasSelfJoinEnabled: GroupSelfJoin,
 )
 
+/**
+ * Represents user's group.
+ *
+ * @param id            the IRI if the group.
+ * @param name          the name of the group.
+ * @param descriptions  the descriptions of the group.
+ * @param project       the project this group belongs to.
+ * @param status        the group's status.
+ * @param selfjoin      the group's self-join status.
+ */
+case class Group(
```

Comment:
**suggestion (non-blocking):** make case classes final where possible

```suggestion
final case class Group(
```

Reply:
Thanks for pointing out. That is by mistake. Will change it in follow-up PR.